.github/workflows/update-test-rules.yml

```diff
@@ -0,0 +1,31 @@
+name: Update Test Rules
+
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  update-branch:
+    name: Update test-rules Branch
+    runs-on: ubuntu-20.04
+    # https://docs.github.com/en/webhooks-and-events/webhooks/webhook-events-and-payloads#issue_comment
+    if: github.event.issue.pull_request && contains(github.event.comment.body, '/update=test-rules')
```

Comment:
Probably need to figure out how to restrict this to the sublime-security org

Reply:
That'll happen through an environment!